deepcell/model_zoo/panopticnet.py

```diff
@@ -48,21 +48,41 @@
 
 
 def __merge_temporal_features(feature, mode='conv', feature_size=256, frames_per_batch=1):
-    if mode == 'conv':
+    """
+
+    Args:
+        feature: Input layer
+        mode (str, optional): Choose from {'conv','lstm','gru'}. Defaults to 'conv'.
+        feature_size (int, optional): Defaults to 256.
+        frames_per_batch (int, optional): Defaults to 1.
+
+    Raises:
+        ValueError: Mode not 'conv', 'lstm' or 'gru'
+
+    Returns:
+        Same as feature: Output of layer following merge of temporal features
+    """
+
+    acceptable_modes = {'conv', 'lstm', 'gru'}
+    if mode.lower() not in acceptable_modes:

```

Comment:
what if mode is not a string? maybe something like:
`if mode and str(mode).lower() not in....`

Same assumption down below in the ValueError

Reply:
Can I eliminate this concern by starting with `mode = str(mode).lower()`?

Reply:
yeah maybe put it in an if block, so if `mode` is `None`, it stays `None`

Maybe `mode = str(mode).lower() if mode else mode`

Reply:
But I think we should handle if `mode is None` because it does that by default currently.